### PR TITLE
[pt-BR] SkiptDefaultLib, SkipLib, SourceMap and SourceRoot

### PR DESCRIPTION
**Pull request by juninhogomes**
Hello @khaosdoctor, @alvarocamillont, @danilofuchs e @orta

In this PR are the files:

> skipDefaultLibCheck.md
> skipLibCheck.md
> sourceMap.md
> sourceRoot.md

**Some notes about:**

> skipLibCheck.md
 
* I Kept the term `type` in English as in the `exclud.md` file, but in DeclarationDir.md` the term was translated. 

    * I did this because there several files that let terms between backticks on the original language.

* I changed the link to Yarn's resolutions, for the pt-BR version.

____

> sourceMap.md

* Changed the link from Sourcemap Files to the pt-PT version on Mozilla Website. They don't have a pt-Br version yet.

____

> sourceRoot.md

* I used "raiz do código-fonte" for the term "sourceRoot" thinking about what the function does. There other terms that can be used, like "Raiz de origem" or something like this, but I guess that this term fits a little better.  Open to suggestions anyway.

____

And that's it for now. Thank you, guys! 😃

### CODE REVIEW

File: packages/tsconfig-reference/copy/pt/options/sourceMap.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+```js
+// helloWorld.js
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.helloWorld = "Olá";
+//# sourceMappingURL=// helloWorld.js.map
+```
+
+E isso irá gerar o seguinte mapa json:
```
Comment (alvarocamillont) [inline packages/tsconfig-reference/copy/pt/options/sourceMap.md]: E isso irá gerar o seguinte arquivo json do mapa

**Review by alvarocamillont — CHANGES_REQUESTED** [review]

File: packages/tsconfig-reference/copy/pt/options/sourceMap.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+display: "Mapas de origem"
+oneline: "Cria mapas de origem para os arquivos JavaScript emitidos"
+
+---
+
+Permite a geração de [mapas de origem](https://developer.mozilla.org/pt-PT/docs/Tools/Debugger/How_to/Use_a_source_map). Esses arquivos permitem a depuradores e outras ferramentas exibir o código fonte TypeScript original ao trabalhar com os arquivos JavaScript emitidos. Mapas de origem são emitidos como arquivos `js.map` (ou `jsx.map`) próximos aos arquivos de saída `.js` correspondentes.
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/sourceMap.md]:
```suggestion
Permite a geração de [mapas de origem](https://developer.mozilla.org/pt-PT/docs/Tools/Debugger/How_to/Use_a_source_map). Esses arquivos permitem que depuradores e outras ferramentas exibam o código fonte TypeScript original ao trabalhar com os arquivos JavaScript emitidos. Mapas de origem são emitidos como arquivos `js.map` (ou `jsx.map`) ao lado dos arquivos de saída `.js` correspondentes.
```

**Comment by juninhogomes** [comment]
@alvarocamillont, @danilofuchs changes in SourceMap made

**Review by alvarocamillont — APPROVED** [review]

**Comment by danilofuchs** [comment]
LGTM

**Comment by alvarocamillont** [comment]
LGTM

**Comment by orta** [comment]
LGTM (I believe I've fixed the action too)

File: packages/tsconfig-reference/copy/pt/options/sourceMap.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+display: "Mapas de origem"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/sourceMap.md]:
```suggestion
display: "Mapas de código"
```

File: packages/tsconfig-reference/copy/pt/options/sourceMap.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+oneline: "Cria mapas de origem para os arquivos JavaScript emitidos"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/sourceMap.md]:
```suggestion
oneline: "Cria mapas de código para os arquivos JavaScript emitidos"
```

File: packages/tsconfig-reference/copy/pt/options/sourceMap.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+---
+
+Permite a geração de [mapas de origem](https://developer.mozilla.org/pt-PT/docs/Tools/Debugger/How_to/Use_a_source_map). Esses arquivos permitem que depuradores e outras ferramentas exibam o código fonte TypeScript original ao trabalhar com os arquivos JavaScript emitidos. Mapas de origem são emitidos como arquivos `js.map` (ou `jsx.map`) ao lado dos arquivos de saída `.js` correspondentes.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/sourceMap.md]:
```suggestion
Permite a geração de [mapas de código](https://developer.mozilla.org/pt-PT/docs/Tools/Debugger/How_to/Use_a_source_map). Esses arquivos permitem que depuradores e outras ferramentas exibam o código fonte TypeScript original ao trabalhar com os arquivos JavaScript emitidos. Mapas de código são emitidos como arquivos `js.map` (ou `jsx.map`) ao lado dos arquivos de saída `.js` correspondentes.
```

File: packages/tsconfig-reference/copy/pt/options/sourceMap.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+Os arquivos `.js` recebem um container com comentários do mapa de origem, indicando para as ferramentas externas onde os arquivos estão. Por exemplo:
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/sourceMap.md]:
```suggestion
Os arquivos `.js` recebem um container com comentários do mapa de código, indicando para as ferramentas externas onde os arquivos estão. Por exemplo:
```

File: packages/tsconfig-reference/copy/pt/options/sourceRoot.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+---
+display: "Raiz do código-fonte"
+oneline: "Configura o caminho da raiz para depuradores acharem o código-fonte referenciado"
+
+---
+
+Especifica a localização onde o depurador deve encontrar os arquivos TypeScript ao invés de outros locais relativos de origem.
+Essa string é processada literalmente dentro do mapa de origem, onde você pode utilizar um caminho ou uma URL:
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/sourceRoot.md:8]:
```suggestion
Essa string é processada literalmente dentro do mapa de fonte, onde você pode utilizar um caminho ou uma URL:
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
I think you can translate `source map` as `mapas de fonte`

@danilofuchs @alvarocamillont what do you think?

**Comment by danilofuchs** [comment]
> I think you can translate `source map` as `mapas de fonte`
> 
> @danilofuchs @alvarocamillont what do you think?

Personally, I think `mapa de origem` makes a little more sense, although it is inconsistent with other translations.

`source maps` is a term that only exists in this specific context. I think the least we should do is write `mapa de origem (source map)` or `mapa de fonte (source map)` in the `display` or `oneline` attributes

**Comment by alvarocamillont** [comment]
I like ' mapa de origem" more too

**Comment by khaosdoctor** [comment]
My thinking on this was due to `source code` being translated to `código fonte`, and `source map` using `source` as `origem` and not `fonte`. What about `mapa de código`?

**Comment by juninhogomes** [comment]
> My thinking on this was due to `source code` being translated to `código fonte`, and `source map` using `source` as `origem` and not `fonte`. What about `mapa de código`?

I like  `mapa de código`, guess that makes sense.

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Updated my suggestions

**Comment by juninhogomes** [comment]
@khaosdoctor Updated

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Comment by khaosdoctor** [comment]
With this PR we merged all files in the tsconfig section <3

@orta the tsconfig section is finished

**Pull request merged**